src/sentry/runner/commands/devservices.py

```diff
@@ -303,6 +303,19 @@ def up(
     """
     from sentry.runner import configure
 
+    click.secho(
+        """
+WARNING: We're transitioning from `sentry devservices` to the new and improved `devservices` in the near future.
+To give the new devservices a try, set the `USE_NEW_DEVSERVICES` environment variable to `1`.
```

Comment:
Should we link to some docs for more information?

Reply:
I don't like the idea of linking a notion page here, so I think the devservices repo readme should suffice. Thoughts?